.gitignore

```diff
@@ -1,3 +1,5 @@
 target/
 */target/
 */*/target/
+.idea
+.DS_Store
```

Comment:
Speaking as a fellow IntelliJ user...

I'd suggest adding the following to your personal `~/.gitconfig` instead:

```
[core]
  excludesfile = /home/myuser/.gitignore
```

...and then putting these kinds of directives in there.

That avoids every other project having to add `.gitignore` directives for every editor.